Sources/Routing/Router.swift

```diff
@@ -26,6 +26,26 @@ public class Router {
     private var _cache: [String: Responder?] = [:]
     private var _cacheLock = NSLock()
 
+    /// Removes the cached Responder for a given Request.
+    /// If there is no cached Responder, returns nil.
+    ///
+    /// NOTE: If you do not register a new Responder for the
+    /// Request, the old Responder will be invoked on a subsequent
+    /// Request and re-cached. I.E. this function does not prune
+    /// the Branch.
+    @discardableResult
+    public func flushCache(for request: Request) -> Responder? {
```

Comment:
Would it make sense to add `flushCache()` as well to flush the entire cache at once?